### PR TITLE
[simple_kit]: Add common action sheet and sheet items

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                          children: [
+                            SizedBox(
+                              width: isCreditCard ? 180.w : 150.w,
+                              child: Baseline(
+                                baseline: 15.4.h,
+                                baselineType: TextBaseline.alphabetic,
+                                child: Text(
+                                  description,
+                                  style: sCaptionTextStyle.copyWith(
+                                    color: SColorsLight().grey3,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:86]: colors pod?

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                            ),
+                            const Spacer(),
+                            if (isCreditCard)
+                              SizedBox(
+                                width: 90.w,
+                                child: Text(
+                                  helperText,
+                                  textAlign: TextAlign.end,
+                                  style: sCaptionTextStyle.copyWith(
+                                    color: SColorsLight().grey3,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:99]: please review pr and fix everywhere to colors pod

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                          ],
+                        ),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+              const Spacer(),
+              SDivider(
+                width: 327.w,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:112]: double.infinity?

File: plugins/simple_kit/lib/src/action_sheet/components/simple_fiat_sheet_item.dart [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+              const Spacer(),
+              SDivider(
+                width: 327.w,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/action_sheet/components/simple_fiat_sheet_item.dart:70]: same here

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart (hunk 3 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:112]: no

File: plugins/simple_kit/lib/src/action_sheet/components/simple_fiat_sheet_item.dart (hunk 4 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/simple_fiat_sheet_item.dart:70]: no

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart (hunk 3 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:112]: we don't need a full width

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart (hunk 1 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:86]: answered in Tg

File: plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart (hunk 2 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/action_sheet/components/simple_asset_sheet_item.dart:99]: answered in Tg

**Pull request merged**